.ddev/config.yaml

```diff
@@ -0,0 +1,182 @@
+name: tea
+type: php
+docroot: ""
+php_version: "7.4"
+webserver_type: nginx-fpm
+router_http_port: "80"
+router_https_port: "443"
+xdebug_enabled: true
+additional_hostnames: []
+additional_fqdns: []
+mariadb_version: "10.5"
+mysql_version: ""
+hooks:
+  post-start:
+  - exec: composer install --no-progress
+use_dns_when_possible: true
+composer_version: ""
+web_environment: []
+
+
+# This config.yaml was created with ddev version v1.17.0
```

Comment:
And in the meantime, ddev 1.17.1 has been released. (I don't know whether this has any relevance to the generated configuration file, though.)